src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -85,15 +88,15 @@ public UserDto updateUserInfo(@Valid @RequestBody UserDto userDto, Principal pri
     @ApiOperation(value = "Add children data for a registered requested user")
```

Comment:
I think you can use a constant here, something like
private static final String MAX_AMOUNT_CHILDREN;

Reply:
Nice approach! 👍